INTERPRETING.md

```diff
@@ -352,3 +352,30 @@ following strings:
 - **`generated`** The test file was created procedurally using the project's
   test generation tool. This flag is specified for informational purposes only
   and has no bearing on how the test should be interpreted.
+
+### `locale`
+
+The `locale` attribute allows tests to declare explicit information regarding locale specificity.
+
+- **`tags`** A list of valid language tags or subtags found in this test. Use IANA Language Subtag Registry to validate.
```

Comment:
"Use IANA Language Subtag Registry to validate." That seems more like instruction to test contributors--do you think it would fit better in CONTRIBUTING.md?

Reply:
I prefer having all the tags presented in the same place, and this phrase is too small to go to contributing as it is. Maybe we should follow up to add a paragraph with more context in that other doc? For that I would not block this current PR.